boefjes/boefjes/plugins/kat_cve_2023_35078/normalize.py

```diff
@@ -0,0 +1,78 @@
+from typing import Iterable, List, Tuple, Union
+
+from boefjes.job_models import NormalizerMeta
+from octopoes.models import OOI, Reference
+from octopoes.models.ooi.findings import CVEFindingType, Finding
+from octopoes.models.ooi.software import Software, SoftwareInstance
+from packaging import version
+
+VULNERABLE_RANGES: List[Tuple[str, str]] = [("0", "11.8.1.1"), ("11.9.0.0", "11.9.1.1"), ("11.10.0.0", "11.10.0.2")]
+
+
+def extract_js_version(html_content: str) -> Union[version.Version, bool]:
+    telltale = "/mifs/scripts/auth.js?"
+    telltale_position = html_content.find(telltale)
+    if telltale_position == -1:
+        return False
+    version_end = html_content.find('"', telltale_position)
```

Comment:
We should probably make sure we dont get -1 for this  end offset. (and the same for the css offset)